libvips/foreign/webpsave.c

```diff
@@ -724,15 +726,17 @@ static const char *vips__save_webp_suffs[] = { ".webp", NULL };
 
 /* Type promotion for save ... just always go to uchar.
  */
+// clang-format off
 static int bandfmt_webp[10] = {
-/* UC  C   US  S   UI  I   F   X   D   DX */
-   UC, UC, UC, UC, UC, UC, UC, UC, UC, UC
+/*  UC  C   US  S   UI  I   F   X   D   DX */
+	UC, UC, UC, UC, UC, UC, UC, UC, UC, UC
```

Comment:
Looks like GitHub's UI is using `TabWidth: 8` by default. Appending `?ts=4` to the URL makes this comment aligned
https://github.com/libvips/libvips/pull/3125/files?ts=4

This can also be controlled in https://github.com/settings/appearance.

Reply:
I suppose that's just a coincidence though --- I think that was aligned with tabwidth 8, and the alignment is thrown off by github adding the leading + char. Anyway, a minor thing!

Reply:
Ah, you're right. Unfortunately, `TabWidth: 4` is still required in `.clang-format` because otherwise it uses spaces for continuation, despite using `UseTab: Always` and/or `ContinuationIndentWidth: 8`. WDYT about commit c2a037b07c20c96560c219ff2005047a66103af2 instead?

We could also consider preferring 4 spaces over tabs, but I don't want to interfere with anyone's workflow.